src/pages/profile/profile.tsx

```diff
@@ -15,67 +17,66 @@ export const ProfilePage: React.FunctionComponent = () => {
           height: 'calc(100vh - 65.3px)',
         }}
       >
-        <Box
-          sx={{
-            display: 'flex',
-            justifyContent: 'flex-end',
-            flexDirection: 'column',
-            width: 1,
-            height: '500px',
-            backgroundColor: 'primary.light',
-            borderRadius: '33px',
-            mt: '200px',
-            mb: '40px',
-          }}
-        >
-          <Box
-            sx={{
-              display: 'flex',
-              justifyContent: 'center',
-              alignItems: 'center',
-              mb: '50px',
-            }}
-          >
-            <Avatar
-              alt="Remy Sharp"
-              srcSet="https://citaty.info/files/styles/poster/public/characters/8604.jpg?itok=_bemMaL1"
-              sx={{
-                width: { sm: '250px', md: '270px', xs: '200px' },
-                height: { sm: '250px', md: '270px', xs: '200px' },
-              }}
-            />
-          </Box>
+        {isAuth ? (
           <Box
             sx={{
               display: 'flex',
+              justifyContent: 'flex-end',
               flexDirection: 'column',
               width: 1,
-              alignItems: 'center',
+              height: '500px',
+              backgroundColor: 'primary.light',
+              borderRadius: '33px',
+              mt: '130px',
+              mb: '40px',
             }}
           >
-            <Typography variant="h4" color="initial">
-              Harry Potter
-            </Typography>
-            <Typography variant="subtitle1" color="initial">
-              Wizard
-            </Typography>
             <Box
               sx={{
                 display: 'flex',
-                justifyContent: 'space-between',
-                p: '10px',
-                mb: '40px',
-                borderRadius: '33px',
+                justifyContent: 'center',
+                alignItems: 'center',
+                mb: '50px',
               }}
             >
+              <Avatar
+                alt="Remy Sharp"
+                srcSet="https://citaty.info/files/styles/poster/public/characters/8604.jpg?itok=_bemMaL1"
+                sx={{
+                  width: { sm: '250px', md: '270px', xs: '200px' },
+                  height: { sm: '250px', md: '270px', xs: '200px' },
+                }}
+              />
+            </Box>
+            <Box
+              sx={{
+                display: 'flex',
+                flexDirection: 'column',
+                width: 1,
+                alignItems: 'center',
+              }}
+            >
+              <Typography variant="h4" color="initial">
+                {user?.nickname}
```

Comment:
you should have user in profile here already

Reply:
fixed, if I understand right. - delete - '? '

Reply:
Object is possibly 'null'. - I thought it didn't matter, I don't render part of page until there is isAuth - not true